model_analyzer/config/generate/run_config_generator.py

```diff
@@ -78,8 +89,47 @@ def set_last_results(self, measurements):
         """
         self._curr_pacg.set_last_results(measurements)
 
-    def _generate_run_config(self, model_config, perf_analyzer_config):
-        run_config = RunConfig(self._model_name, model_config,
+    def _generate_run_config(self, model_configs, perf_analyzer_config):
+        # MM-PHASE 0: Assuming that all models are identical, so using first model's server env
+        run_config = RunConfig(self._model_name, model_configs,
                                perf_analyzer_config,
-                               self._model.triton_server_environment())
+                               self._models[0].triton_server_environment())
```

Comment:
capture triton_server_environment up in line ~44 so there is only one place we need to put this MM-PHASE comment

Reply:
Done